### PR TITLE
Add GIF to README and docs

### LINKED ISSUE #1850: CLI: Add GIF to README
## Description

We should add a GIF showing some functionality of the cli wallet to the README. Devs seem to love those and I do too 😆 

## Open questions (optional)

We could use [terminalizer](https://github.com/faressoft/terminalizer)

## Are you planning to do it yourself in a pull request?

Yes

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added a GIF to the README and the welcome page of the docs. Tell me if you like it.

## Links to any relevant issues

Fixes iotaledger/wallet.rs#1850 

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Thoralf-M** [comment]
I think it's a little bit too fast for that many commands, hard to follow what's going on

**Comment by Dr-Electron** [comment]
I can slow it down or use rl speed 🤔 
Maybe I could even upload the config itself. It's basically just a yaml file with config and an array of delay and chars to specifiy what was entered when. Pretty cool. Just think that's not fitting into the repo 😅

**Comment by thibault-martinez** [comment]
Yeah it's a bit too fast, or maybe just add a pause at the end so people can read everything?

**Comment by Dr-Electron** [comment]
@thibault-martinez slowed down the last frame. Wdyt?

(I hope the GIF is still up-to-date 😅 )

**Comment by Thoralf-M** [comment]
Can you read everything so fast? I can't

**Comment by thibault-martinez** [comment]
Yeah I'd say bit too fast. Better too slow than too fast.

**Comment by Dr-Electron** [comment]
> or maybe just add a pause at the end so people can read everything?

:trollface: 

I just went the lazy route first ;). Will make it slower

**Comment by Dr-Electron** [comment]
I love how you can check the git changes and directly compare the animation :)

So what about now?

**Review by Thoralf-M — DISMISSED** [review]
The balance at the end is still a lot with not much time to read, but okay

**Comment by Dr-Electron** [comment]
Just tell me how many seconds you want to wait at the end. Or if you want to slow it down even more in general.
Currently all frames have a delay of 150ms and the last a delay of 1200ms.
We could even stop the "looping" after one run 🤔

**Comment by Thoralf-M** [comment]
I think the first frames are fine, can you maybe do the latest one 3000ms? Or yeah stop it completely, if that's not too boring for you then :trollface:

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]
